source/nodejs/tests/unit-tests/template-test-resources/version-template.json

```diff
@@ -0,0 +1,21 @@
+{
+    "type": "AdaptiveCard",
+    "version": "1.0",
+    "body": [
+        {
+            "$when": "${$_acTemplateVersion.major >= 2}",
+            "type": "TextBlock",
+            "text": "$_acTemplateVersion.major >= 2"
+        },
+        {
+            "$when": "${$_acTemplateVersion.minor >= 0}",
+            "type": "TextBlock",
+            "text": "$_acTemplateVersion.minor >= 0"
+        },
+        {
+            "$when": "${$_acTemplateVersion.patch >= 0}",
+            "type": "TextBlock",
+            "text": "$_acTemplateVersion.patch >= 0"
+        }
```

Comment:
it could be nice to have some conditions that fail as well:)

Reply:
Since we pull the version directly from `package.json`, how would we force a failing condition?

Reply:
Oh, I just mean something like
` "$when": "${$_acTemplateVersion.patch < 0}"` - to make sure that `TextBlock` is not rendered in this case :)

Reply:
@paulcam206 @jwoo-msft it would also be nice if we could set an arbitrary version in `package.json` (or some other way) to test proper handling of improperly structured version string :) but I assume that would be unlikely?

Reply:
I just added two new unit tests :)